### PR TITLE
Add 0.19.3 change

### PR DESCRIPTION
**Pull request by d8660091**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -30,6 +30,22 @@ description: >
...
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.19.2 |    ✔    |     ✔      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     —      |    ✔    |     ✔      |  —   |
+
+### Changed
+- Updated helm to v3.14.3 [#3050](https://github.com/aws/eks-anywhere-build-tooling/pull/3050)
+
+### Fixed
+- Bumped golang.org/x/net that has a fix for [vunerability GO-2024-2687](https://pkg.go.dev/vuln/GO-2024-2687)
```
Comment (ahreehong) [inline docs/content/en/docs/whatsnew/changelog.md:49]: - Fixed proxy configurations for airgapped enviornments [#7913](https://github.com/aws/eks-anywhere/pull/7913)

File: docs/content/en/docs/whatsnew/changelog.md [hunk 2]
```diff
@@ -30,6 +30,23 @@ description: >
...
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.19.2 |    ✔    |     ✔      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     —      |    ✔    |     ✔      |  —   |
+
+### Changed
+- Updated helm to v3.14.3 [#3050](https://github.com/aws/eks-anywhere-build-tooling/pull/3050)
+- Fixed proxy configurations for airgapped enviornments [#7913](https://github.com/aws/eks-anywhere/pull/7913)
```
Comment (ahreehong) [inline docs/content/en/docs/whatsnew/changelog.md:46]: could we move the fixed proxy line under "fixed" instead of "changed"?

**Review by ahreehong — APPROVED** [review]

File: docs/content/en/docs/whatsnew/changelog.md [hunk 3]
```diff
@@ -30,6 +30,23 @@ description: >
...
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.19.2 |    ✔    |     ✔      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     —      |    ✔    |     ✔      |  —   |
+
+### Changed
+- Updated helm to v3.14.3 [#3050](https://github.com/aws/eks-anywhere-build-tooling/pull/3050)
+
+### Fixed
+- Bumped golang.org/x/net that has a fix for [vunerability GO-2024-2687](https://pkg.go.dev/vuln/GO-2024-2687)
```
Comment (abhay-krishna) [inline docs/content/en/docs/whatsnew/changelog.md:48]:
```suggestion
- Bumped golang.org/x/net that has a fix for [vulnerability GO-2024-2687](https://pkg.go.dev/vuln/GO-2024-2687)
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 4]
```diff
@@ -30,6 +30,23 @@ description: >
...
+- Fixed proxy configurations for airgapped enviornments [#7913](https://github.com/aws/eks-anywhere/pull/7913)
```
Comment (abhay-krishna) [inline docs/content/en/docs/whatsnew/changelog.md:49]:
```suggestion
- Fixed proxy configurations for airgapped environments [#7913](https://github.com/aws/eks-anywhere/pull/7913)
```

**Pull request merged**